feeder/parsers/PL.py

```diff
@@ -30,6 +32,7 @@ def fetch_PL():
 
     data = {
         'countryCode': COUNTRY_CODE,
+        'datetime': date,
```

Comment:
The datetime required here is the datetime of the measurement. So you need to get read it from the xml. I think you can calculate it from the "position" key that corresponds to the "quantity" value, but not sure.